containers/proxy-squid-image/Dockerfile

```diff
@@ -65,4 +65,4 @@ EXPOSE 8080/tcp
 
 VOLUME [ "/etc/uyuni", "/var/cache/squid" ]
 
-CMD uyuni-configure.py && squid -z --foreground && squid -FC -d 1 --foreground
+CMD uyuni-configure.py && squid -z --foreground && squid -FC -d 1 --foreground --enable-delay-pools
```

Comment:
Do we want to enable this option for everyone?

Reply:
We could enable it conditionally I think, if the bandwidth limit is set. Let's see